src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java

```diff
@@ -189,6 +189,8 @@
 
     @NotBlank(groups = {ApplicationIntestacyGroup.class}, message = "{solsMultipleClaimsIsNull}")
     private final String solsMultipleClaims;
+    
+    private final String solsApplicantSiblings;
```

Comment:
we will also need this in the CallbackResponseTransformer.java and we should add a test for this field in the CallbackResponseTransformerTest.java